src/plugins/intel_npu/src/plugin/npuw/partitioning/online/compiler.cpp

```diff
@@ -410,7 +415,11 @@ class Compiler {
 
         switch (currentPipeline()) {
         case Pipeline::NONE:
-            break;  // unreachable
+            // Allow partitioning to merge everything into a single group
+            ctx.min_graph_size = 1;
```

Comment:
To be honest, I expect this `ctx.min_graph_size` set in `none()` call

Reply:
With `ctx` in `setCtx()`

Reply:
`none()`, `rep()`, etc only set pass logic and order. `ctx` manipulations are done exactly here for all pipelines (isolates, avoids, nofold, etc). So I would like to keep this logic